pkg/gooq/expression.go

```diff
@@ -36,15 +33,17 @@ type Expression interface {
 	// https://www.postgresql.org/docs/12/sql-expressions.html
 	Filter(...Expression) Expression
 
-	// IMPORTANT: this is for internal use only.
-	// original returns a reference to the original expression. This allows us to recover
-	// an expression's original type. e.g. when we call TableImpl.column1.Eq(String("foo")
+	// IMPORTANT: these are for internal use only.
+	getExpressions() []Expression
+	getOperator() Operator
+	// getOriginal returns a reference to the getOriginal expression. This allows us to recover
+	// an expression's getOriginal type. e.g. when we call TableImpl.column1.Eq(String("foo")
 	// a newBooleanExpression is created with the operands TableImpl.column1 and
 	// String("foo"). inArray the new boolean expression TableImpl.column1 is stored as a
-	// stringExpressionImpl and has lost its original stringFieldImpl type. When
+	// stringExpressionImpl and has lost its getOriginal stringFieldImpl type. When
```

Comment:
There are a few places in the comments that say `getOriginal` instead of just the word `original` now :)

Reply:
Ah! good catch. The GoLand refactoring tool changed the comments also :|